File: pertemuan2/Mahasiswa.java
public class Mahasiswa {
    public static void main(String[] args) {
        TestData mhs = new TestData();
        // input data
        mhs.nama = "Dennis Ritchie";
        mhs.jenisKelamin = 'L';
        mhs.alamat = "Sukoharjo";

        // tampil data
        System.out.println("nama mhs : "+mhs.nama);
        System.out.println("jenis kelamin mhs : "+mhs.jenisKelamin);
        System.out.println("alamat mhs : "+mhs.alamat);
    }
}

class TestData {
    private String nama;
    private char jenisKelamin;
    private String alamat;

    static final int NUM = 10;
}

File: pertemuan2/Mahasiswa.cpp
#include <iostream>

using namespace std;

class Mahasiswa{
private :
    string nama;
    char jenisKelamin;
    string alamat;

public :
    void isiData(){
        nama = "Dennis Ritchie";
        jenisKelamin = 'L';
        alamat = "Sukoharjo";
    }

    void setNama(string mNama){
        nama = mNama;
    }

    string getNama(){
        return nama;
    }

    void setJenisKelamin(char mJenisKelamin){
        jenisKelamin = mJenisKelamin;
    }

    char getJenisKelamin(){
        return jenisKelamin;
    }

    void setAlamat(string mAlamat){
        alamat = mAlamat;
    }

    string getAlamat(){
        return alamat;
    }
};

int main(){
    Mahasiswa mhs;
    mhs.isiData();
}